pom.xml

```diff
@@ -24,14 +23,14 @@
   </scm>
   <properties>
     <version.accumulo>2.1.1</version.accumulo>
-    <version.avro>1.8.2</version.avro>
-    <version.datawave>5.11.6</version.datawave>
+    <version.avro>1.11.3</version.avro>
+    <version.datawave>6.5.1</version.datawave>
     <version.guava>25.1-jre</version.guava>
-    <version.hadoop>3.3.3</version.hadoop>
-    <version.microservice.accumulo-utils>2.1.1</version.microservice.accumulo-utils>
+    <version.hadoop>3.3.4</version.hadoop>
+    <version.microservice.accumulo-utils>2.1.2</version.microservice.accumulo-utils>
     <version.microservice.ingest.common>${project.version}</version.microservice.ingest.common>
     <version.microservice.metadata-utils>1.9</version.microservice.metadata-utils>
-    <version.microservice.starter>2.0.2-2.7.1</version.microservice.starter>
+    <version.microservice.starter>3.0.0</version.microservice.starter>
```

Comment:
Need to get this version published and accessible: https://github.com/NationalSecurityAgency/datawave/packages/2049002